src/utils.js

```diff
@@ -177,6 +177,7 @@ export function populateQuery (valueObj, query, startPath = '') {
  * @returns {Boolean} true if valid
  */
 export function hasValidQueryValues (value, queryDef, startPath) {
+  // need to invert this for the next major rev since returning true is contrary to the function name
```

Comment:
I guess it depends on how you want this to behave. When I read it, my first thought was that we're checking to see if it have dependencies that resolve properly. If they have no query dependencies at all, then wouldn't it have no query values. This came up when I was trying to prevent the `formValueChange` from re-rendering this input and I wanted to separate the check for the queries and the check for the *uninitialized* `enum` type.